docs/dev/Pagination-v2.md

```diff
@@ -114,67 +496,75 @@ sequenceDiagram
     participant CreatePagingTableScanBuilder
     participant OpenSearchExecutionEngine
     participant PlanSerializer
-    participant Physical Plan Tree
 
-SQLService->>+QueryPlanFactory: execute
-  critical
-  QueryPlanFactory->>+CanPaginateVisitor: canConvertToCursor
-    CanPaginateVisitor-->>-QueryPlanFactory: true
+SQLService ->>+ QueryPlanFactory : execute
+  rect rgb(191, 223, 255)
```

Comment:
Change color, it is not readable on dark GH theme
![image](https://github.com/opensearch-project/sql/assets/88679692/c7f0ef52-a4bb-43e8-9a3e-1d3af8ed7fbc)

Reply:
Fixed in d3888ceaf